src/main/resources/static/stylesheets/style.css

```diff
@@ -49,4 +49,16 @@
 
 .navbar-option {
     box-shadow: 0 0 1px 0 #33332a;
+}
+
+.form-div {
+    width: 1080px;
+}
+
+.synch-width{
+    width: 864px
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a semicolon at the end of this declaration. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=css%3Asemicolon-declaration)